tiledb/fragment.py

```diff
@@ -12,6 +12,86 @@
 class FragmentInfoList:
     """
     Class representing an ordered list of FragmentInfo objects.
+
+    :param uri: URIs of fragments
+    :param version: Fragment version of each fragment
+    :param nonempty_domain: Non-empty domain of each fragment
+    :param cell_num: Number of cells in each fragment
+    :param timestamp_range: Timestamp range of when each fragment was written
+    :param dense: For each fragment, True if fragment is dense, else False
+    :param sparse: For each fragment, True if fragment is sparse, else False
+    :param has_consolidated_metadata: For each fragment, True if fragment has consolidated fragment metadata, else False
+    :param unconsolidated_metadata_num: Number of unconsolidated metadata fragments in each fragment
+    :param to_vacuum_num: Number of already consolidated fragments to vacuum
+    :param to_vacuum_uri: URIs of already consolidated fragments to vacuum
+
+    **Example:**
```

Comment:
This might just be a local issue, but the examples here are not rendering for me.